src/Application/Controller/Projects.php

```diff
@@ -80,14 +80,18 @@ public function profiles() {
 					$this->project->removeEncodingProfileVersion(
 						$values['remove']
 					);
-				} elseif (!empty($values['add'])) {
+				} elseif (!empty($values['add']) and
+					is_array($values['add'])) {
 					$this->project->save([
 						'EncodingProfileVersion' => [
-							['encoding_profile_version_id' => $values['add']]
+							[
+								'encoding_profile_version_id' => $values['add']['version'],
+								'priority' => $values['add']['priority'],
+								'auto_create' => $values['add']['auto_create']
+							]
```

Comment:
When `$values['add']` is an array, it should work to change `version` in the template to `encoding_profile_version_id` and just use

```php
$this->project->save([
	'EncodingProfileVersion' => [
		$values['add']
	]
]);
```